.config/base.js

```diff
@@ -0,0 +1,26 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import typescript from 'rollup-plugin-typescript2';
+
+const envHotType = process.env.HOT_TYPE;
+
+export const baseConfig = {
+  input: 'src/common/index.tsx',
+  plugins: [
+    replace({
+      'hot-alias': envHotType === 'pro' ? 'handsontable-pro' : 'handsontable',
```

Comment:
Maybe `process.env.HOT_TYPE` could be just `handsontable-pro` or `handsontable`?

Reply:
Why?